cypress/support/fragments/circulation-log/searchPane.js

```diff
@@ -11,6 +11,13 @@ export default {
     cy.get('[class^="button-"][type="submit"]').first().click();
   },
 
+  searchByUserBarcode(barcode) {
+    cy.do([
+      TextField({ name: 'userBarcode' }).fillIn(barcode),
+    ]);
+    cy.get('[class^="button-"][type="submit"]').first().click();
```

Comment:
why so difficult?

Reply:
Because we have 2 identical buttons, but first section without name, id or another selector =(
![image](https://user-images.githubusercontent.com/26041116/159974663-99c486e4-363d-4d23-b4df-9a3411b23aef.png)